src/main/java/seedu/address/model/person/UserProfile.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.model.enumeration.FocusArea;
+import seedu.address.model.enumeration.Major;
+
+
+//@@author RomaRomama
+
+/**
+ * Presents profile of User
+ */
+
+public class UserProfile {
```

Comment:
Can you remove this `UserProfile` class in your PR? `AddCommand` does not really need `UserProfile` but my PR (`SetupCommand`) needs it, so putting it in your PR prevents me from working on my PR.